src/services/local-database-service.ts

```diff
@@ -31,10 +31,23 @@ export default class LocalDatabaseService {
     return localDB.tours.get(id)
   }
 
+  public async markTourAsDeleted (tour: Tour): Promise<void> {
+    const localTour = await localDB.tours.get(tour.id)
+    if (localTour) {
+      localTour.isDeleted = 1
+      localTour.isSynced = 0
```

Comment:
@Saela I think we shoud add a note somewhere that we have to resort to this... ugly workaround due to 3rd party limitations.

Reply:
Ah, follows below, never mind :D